backend/src/main/java/coe/datacollection/User.java

```diff
@@ -36,8 +36,8 @@ public class User {
     @Column(name = "pin")
     private char[] pin;
 
-    @Column(name = "salt")
-    private char[] salt;
+    //@Column(name = "salt")
+    //private char[] salt;
```

Comment:
removing salt because it is not used in the login branch

allows the removal of the field from the database also